.github/ISSUE_TEMPLATE/1.bug.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report 🐛
+about: Create a bug report for Material-UI.
+labels: 'status: needs triage'
+---
+
+<!-- Provide a general summary of the issue in the Title above -->
+
+<!--
+  Thank you very much for contributing to Material-UI by creating an issue! ❤️
+  To avoid duplicate issues we ask you to check off the following list.
+-->
+
+<!-- Checked checkbox should look like this: [x] -->
+
+- [ ] The issue is present in the latest release.
+- [ ] I have searched the [issues](https://github.com/mui-org/material-ui-pickers/issues) of this repository and believe that this is not a duplicate.
+
+## Current Behavior 😯
+
+<!-- Describe what happens instead of the expected behavior. -->
+
+## Expected Behavior 🤔
+
+<!-- Describe what should happen. -->
+
+## Steps to Reproduce 🕹
+
+<!--
+  Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
+  Include code to reproduce, if relevant (which it most likely is).
+
+  You should use the official codesandbox template as a starting point: https://codesandbox.io/s/gifted-wilbur-g4bgc
```

Comment:
```suggestion
  You should use the official codesandbox template as a starting point: https://codesandbox.io/s/material-ui-pickers-v4-jfm6g
```

Reply:
Ah right, let's add a new team template for bugs related to the date picker. Did you receive the invitation email?

This would avoid a downstream issue when we add more arguments to the onChange callback, as well as being more consistent with the other components.

<img width="748" alt="Capture d’écran 2020-06-24 à 17 58 50" src="https://user-images.githubusercontent.com/3165635/85590153-767a9d80-b644-11ea-8dbc-4cbd2ff0c1d5.png">

Regarding the template itself, I think that we should do:

```diff
- const [selectedDate, handleDateChange] = useState(new Date());
+ const [value, setValue] = useState(new Date());

  return (
    <LocalizationProvider dateAdapter={DateFnsUtils}>
      <DatePicker
-       value={selectedDate}
+       value={value}
-       onChange={handleDateChange}
+       onChange={(newValue) => {
+         setValue(newValue);
        }}
        renderInput={props => <TextField {...props} />}
      />
```

Reply:
Didn't receive the invitation

Reply:
Invited.

I have continued to improve the template in: remove date-io dependency, remove fixed version dependency, update useState to fall into the nominal use case. https://codesandbox.io/s/material-ui-date-picker-ppt05

Reply:
I am not sure when can I find the invitation, no email and not seeing anything in the UI :(